static/elements/chromedash-feature.html

```diff
@@ -165,7 +165,7 @@ <h3>Chromium status</h3>
               <span>[[feature.browsers.chrome.webview]]</span>
             </span>
           </template>
-          <template is="dom-if" if="[[!feature.browsers.chrome.origintrial]]">
+          <!-- <template is="dom-if" if="[[!feature.browsers.chrome.origintrial]]">
```

Comment:
Think you still want to keep this outer conditional. Just comment out the inner opera bits.

Reply:
The inner Opera is the only thing inside this particular tag which is why I its included. The closing template is on line 192.

Reply:
I see now. Wasn't reading the diff correctly.